package.json

```diff
@@ -71,6 +72,22 @@
             "maximum": 512,
             "description": "%hexeditor.columnWidth%"
           },
+          "hexeditor.copyType": {
```

Comment:
It'd be cool to also have button in the "Copy As..." pick (`QuickPick.buttons`) to bring the user to this setting.

you can do so with `vscode.commands.executeCommand('workbench.action.openSettings2', { query: '@id:hexeditor.copyType' })`

Reply:
added by [7d5dfde](https://github.com/microsoft/vscode-hexeditor/pull/540/commits/7d5dfde8c5ba5a1e6900fea6bcba54c6f7dd9d41)